### ISSUE TITLE
Mod does not activate. Installed correctly(as far as I can tell) and other mods work fine, but VA2 will not 'open' when I click it.

### ISSUE THREAD

**Issue by AtmaDarkwolf** [BarleyFlour/VisualAdjustments2#77]
as the title, mod simply does not activate. Unsure if cuz hasn't been updated or not(which is likely) but since see nobody else mentioning it I felt it may be unknown.

**Comment by sunseekers0905** [comment]
Yeah, same here. It is just not activating even though the 'Visuadl Adjustments2' isclearly up and running aon 'OK' on Unity Mod

**Comment by maboroshix** [comment]
Yes, just adding to the chorus here -- not working as of the update on June 28, 2024 to 1.2.7. I noticed it when I looted a Flameguard shield and it was a textureless pink and not situated properly on the arm of the character. The tabs for 'Visual' and all the related tabs show up but are all blank. Just further info for the author, hope it helps. Really like the mod so I am hoping it gets fixed relatively soon!

**Comment by Smallgiant801** [comment]
Same issue, could there be a mod conflict? 
I have toy box and custom portrait.

**Comment by Smallgiant801** [comment]
Solved. It doesn't work like it used to. Open up the inventory then look all the way to the right new section called visuals should be there.

**Comment by sunseekers0905** [comment]
Looking at it now.  Am in inventory but there is no new section over to the right. :-(  The last tab on the right is 'Map'
